frontend/src/components/Thread.vue

```diff
@@ -53,16 +69,34 @@ export default {
     },
   },
   methods: {
+    // TODO: this is currently not used, because I can't figure out a good place to put this
     resolve(thread) {
       this.$store.commit('resolveThread', thread);
     },
     reply(thread) {
-      console.log(thread);
-      this.$store.commit('addMessageToThread', [this.newMessages[thread.id], thread]);
-      this.newMessages[thread.id] = "";
+      if (this.newMessages[thread.id]) {
+        console.log(thread);
+        this.$store.commit('addMessageToThread', [this.newMessages[thread.id], thread]);
+        this.newMessages[thread.id] = "";
+      } else {
+        console.log("Reply is empty!");
```

Comment:
same as comment for other console.log

Reply:
This one is needed, just to we know something is happening from the developer's side. This can be removed once we have alerts though.